analyzer/src/test/kotlin/managers/utils/MavenSupportTest.kt

```diff
@@ -30,6 +30,27 @@ import org.ossreviewtoolkit.model.VcsInfo
 import org.ossreviewtoolkit.model.VcsType
 
 class MavenSupportTest : WordSpec({
+    "getOriginalScm()" should {
+        "return the parent's SCM connection if the child SCM uses the parent's SCM connection implicitly" {
+            val mavenProject = MavenProject().apply {
+                scm = Scm().apply {
+                    connection = "https://github.com/spring-projects/spring-boot/childArtifactName"
```

Comment:
Even if it does not matter for this test, please use the correct semantics for the `connection` field and prefix it with `scm:git:` and use the real clone URL that ends in `.git`. That way it also can't be mixed up with the `url` field (like right now in line 42 and 43).

Reply:
When using these semantics, would it look like this?

The child gets an SCM connection from Maven like this:
`scm:git:https://github.com/spring-projects/spring-boot/childArtifactName.git`
if the parent uses:
`scm:git:https://github.com/spring-projects/spring-boot.git`

If this is how it works, the fixup does no longer work.

I guess it is time to prepare an example project and to write a funTest to find out.

Reply:
Ok, I tested it, and Maven really uses this SCM connection for the child:
```
scm:git:https://example.com/git/repository.git/maven-app
```